CONTRIBUTING.md

```diff
@@ -194,6 +194,21 @@ Commit messages should never contain references to any of:
       https://github.com/google/zerocopy/commit/789b3deb)
 1. Other entities which may not make sense to arbitrary future readers
 
+### Git hooks
+
+There is a pre-push git hook available at ./githooks/pre-push .  This hook runs
```

Comment:
```suggestion
There is a pre-push git hook available at `./githooks/pre-push`. This hook runs
```

Single period between sentences to be consistent w/ other documentation. Can you do the same in the rest of this added section?